client/src/App.js

```diff
@@ -69,6 +47,14 @@ function App() {
     window.URL.revokeObjectURL(url);
   }
 
+  const handleDownloadButtonClick = (event) => {
+    fetch(`${api_endpoint}download_csv?entity_json=${query}&results_json=${JSON.stringify(items)}`)
```

Comment:
I think the max length of a URL is 6000 chars, which we'd hit if `items` has >600 chars per item on average. Alternatively: should we turn this into a POST that has a JSON body which increases our headroom to 1GB?